src/runtimes/node/in_source_config/index.ts

```diff
@@ -97,7 +97,12 @@ export const findISCDeclarations = (
   }
 
   const iscExports = handlerExports
-    .map(({ args, local: exportName }) => {
+    .map((exp) => {
+      if (exp.type !== 'call-expression') {
```

Comment:
just as a note to myself: limiting this to `call-expression` here is fine, because it's not the immediate `export` node, but the resolved value that's being exported. so the code from https://docs.netlify.com/functions/scheduled-functions/ will still work with this change:

```ts
import type { Handler, HandlerEvent, HandlerContext } from "@netlify/functions";
import { schedule } from "@netlify/functions";

const myHandler: Handler = async (event: HandlerEvent, context: HandlerContext) => {
  console.log("Received event:", event);

    return {
        statusCode: 200,
    };
};

const handler = schedule("@hourly", myHandler)

export { handler };```

Reply:
That's right! I've pushed https://github.com/netlify/zip-it-and-ship-it/pull/1530/commits/389445509c91344a3951f60b7edc1aa180535e04 with a couple of tests to prove that.

Reply:
Wait, something is broken ⏳

Reply:
Ah, the tests were failing because this doesn't work in CJS, only the ESM counterpart:

```js
const { schedule } = require("@netlify/functions")

const handler = schedule("@daily", () => {})

exports.handler = handler
```

But this also doesn't work in current `main` because we only get `getExportsFromBindings` on the ESM flow, for some reason: https://github.com/netlify/zip-it-and-ship-it/blob/9c24f23ea2acf2cc258914f37a5491c89107cf81/src/runtimes/node/parser/exports.ts#L83.

We can fix that, but it's neither urgent nor part of this PR. I've skipped that new test for now.